base_custom_info/README.rst

```diff
@@ -1,6 +1,6 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
-    :alt: License: AGPL-3
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
 
 ================
 Base Custom Info
```

Comment:
Fixed in https://github.com/OCA/server-tools/pull/492/commits/be58eb8e63b43e2c7b176662b920c4281002da26.